### DISCUSSION TITLE
Dinit executable

### DISCUSSION THREAD

**Discussion by jjpb44** [davmac314/dinit#202] [Q&A] [answered]
Sorry for the noob question, where is the dinit executable? I was trying to test it but I couldn't find anything similar to an executable in the package (diniti 0.16.1) of the releases. Or the program has to be build?

**Comment by mobin-2008** [comment, marked as answer]
Hi. Dinit doesn't have pre-built executables. You need to get the source and compile it for yourself.

Regards
